README.md

```diff
@@ -88,27 +88,31 @@ For API documentation go to
 
  * [Realm Dart API Docs](https://pub.dev/documentation/realm_dart/latest/)
 
-For a complete documentation go to [Realm Flutter and Dart SDK Docs](https://docs.mongodb.com/realm/sdk/flutter/).
+Use [realm](https://pub.dev/packages/realm) package for Flutter and [realm_dart](https://pub.dev/packages/realm_dart) package for Dart standalone applications.
 
+For complete documentation of the SDKs, go to the [Realm SDK documentation](https://docs.mongodb.com/realm/sdk/flutter/).
 
-## Limitations
+If you are using the Realm SDK for the first time, refer to the [Quick Start documentation](https://www.mongodb.com/docs/realm/sdk/flutter/quick-start/).
+
+To learn more about using Realm with Atlas App Services and Device Sync, refer to the following Realm SDK documentation:
 
-* It provides the functionality for creating, retrieving, querying, sorting, filtering, updating Realm objects.
+- [App Services Overview](https://www.mongodb.com/docs/realm/sdk/flutter/app-services/)
+- [Device Sync Overview](https://www.mongodb.com/docs/realm/sdk/flutter/sync/)
 
-* Migrations are not supported yet.
+## Limitations
 
-    If you change your data models often and receive a migration exception be sure to delete the old `default.realm` file in your application directory. It will get recreated with the new schema the next time the Realm is opened.
+* Realm Database data types Set and Map are not supported yet.
 
 # Realm Flutter SDK
 
-The Realm Flutter package name is `realm`
+The Realm Flutter SDK's package name is [realm](https://pub.dev/packages/realm).
 
 ## Environment setup for Realm Flutter
 
-* Supported platforms are Flutter (iOS, Android, Windows, MacOS) and Dart standalone (Windows, MacOS and Linux)
+* The Realm Flutter SDK supports the platforms iOS, Android, Windows, MacOS, and Linux.
 
-* Flutter ^3.0
-* For Flutter Desktop environment setup check the guide [here](https://docs.flutter.dev/desktop)
+* Flutter ^3.0.3
```

Comment:
In general we should not be mentioning minor versions unless needed as the case with the Dart bug. Also in master we merged a PR which allows `^3.0.0`

Reply:
Yes, probably I will have to fix it in the main branch to be also 3.0.3.

Reply:
Added PR https://github.com/realm/realm-dart/pull/1074